plugins/woocommerce/src/Internal/Logging/RemoteLogger.php

```diff
@@ -324,6 +337,9 @@ protected function is_third_party_error( string $message, array $context ): bool
 			}
 		}
 
+		if ( ! function_exists( 'apply_filters' ) ) {
```

Comment:
I wasn't very sure if we should also wrap the `apply_filters` function because we have doc comments for apply_filters. I didn't wrap it because it will not show lint errors for SafeGlobalFunctionProxy::apply_filters, and if there are some automatically generated docs for apply_filters, it will not include them. Just bringing this up for discussion

Reply:
I think its valid and we can leave this one unwrapped, good thinking!